src/app/checkout/checkout.component.js

```diff
@@ -63,6 +64,13 @@ class CheckoutController{
     });
   }
 
+  signedOut( event ) {
```

Comment:
This seems like a bit of code to have everywhere especially since there are tests that go along with it. Could we maybe make a function in the session service that looks something like this that each page could call:
`function redirectOnSignOut($rootScope, destination){...}`
Maybe the service can inject $rootScope directly... I'm not sure if it's available.